components/camera/replaypcd/replaypcd.go

```diff
@@ -208,9 +304,13 @@ func (replay *pcdCamera) Stream(ctx context.Context, errHandlers ...gostream.Err
 	return stream, errors.New("Stream is unimplemented")
 }
 
-// Close stops replay camera and closes its connections to the cloud.
+// Close stops replay camera, closes the channels and its connections to the cloud.
 func (replay *pcdCamera) Close(ctx context.Context) error {
+	replay.activeBackgroundWorkers.Wait()
```

Comment:
It looks like this wasn't actually used so I've removed it for now. Since Close acquires the write lock it may wait on Reconfigure or NextPointCloud. NextPointCloud shouldnt block for too long since it has a timeout on the download and Reconfigure only sets fields and constructs a cloud connection.